links/README.md

```diff
@@ -1,6 +1,7 @@
 # Tree Integration: Links
 
 A [UI Hook](https://treedocs.now.sh/docs/v1/hooks/ui/introduction/) serving a page that [links](https://treedocs.now.sh/docs/v1/blocks/basic/#link) to another page.
+We provide two examples: an `index.js` demonstrating basic links, and an `index2.js` that shows links with parameters.
```

Comment:
I'd put everything in `index.js`. Page 1 has two links: the first is a simple link to Page 2, the second is a link to Page 3 with params.

Reply:
Got it; fixed!